### ISSUE TITLE
Incorrect speech text by "Horde Scout"

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5445]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Horde scouts at Warsong Lumber Camp sometimes says "Feel the power of the Dark Iron Dwarves!" when you engage them in battle as an alliance character.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Just run past or engage some Horde Scouts at Warsong Lumber Camp, one of them will eventually say this
2. 
3. 

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/39622802-0cec2190-4fc6-11e8-8407-6c732dec1faf.png)
Refer to the chatbox history in the screenshot for proof.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/4515
